modules/javafx.controls/src/main/java/javafx/scene/control/TableCell.java

```diff
@@ -303,9 +303,11 @@ private void setTableView(TableView<S> value) {
     @Override public void startEdit() {
         final TableView<S> table = getTableView();
         final TableColumn<S,T> column = getTableColumn();
-        if (! isEditable() ||
-                (table != null && ! table.isEditable()) ||
-                (column != null && ! getTableColumn().isEditable())) {
+        final TableRow<S> row = getTableRow();
+        if (!isEditable() ||
+                (table != null && !table.isEditable()) ||
+                (column != null && !column.isEditable()) ||
+                (row != null) && !row.isEditable()) {
```

Comment:
Incorrect Line ---- "(row != null) && !row.isEditable())"
Correction required ----  "(row != null && !row.isEditable()))"

Refer similar line which is rightly implemented in TreeTableCell.java.

Reply:
darn .. you certainly have the better eyes :)))

Reply:
Hmm .. this should have been be found by a test ..  having a combination table == null, column == null, row == null would throw. But then, test coverage is .. ;)

Reply:
Good catch, I really don't know how I couldnt see that. 😃 
fixed.